site/content/docs/5.2/components/breadcrumb.md

```diff
@@ -78,6 +78,13 @@ $breadcrumb-divider: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/
 
 You can also remove the divider setting `--bs-breadcrumb-divider: '';` (empty strings in CSS custom properties counts as a value), or setting the Sass variable to `$breadcrumb-divider: none;`.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This variant can not be used because it does not respect the Orange Design System specifications.
+
+For the correct design, please refer to the [breadcrumb](https://system.design.orange.com/0c1af118d/p/774477-local-headers/b/743cd0/i/66611057) component on the Orange Design System website.
```

Comment:
TODO : replace as proposed below.
Please refer to the guidelines...
---
Please refer to the [breadcrumb]() guidelines on the Orange Design System website.

Please refer to the Boosted [Dropdowns]() component. Be aware that this Dropdowns component, because using an `<ul>` tag, can not be a replacement here for technical and accessibility reasons.

For other usages, consider using our Boosted [Tags]() component. You can also refer to the [Tags]() guidelines on the Orange Design System website.

Instead, consider using our Boosted [Checks](), [Radios]() or [Radio toggle buttons]() components.

Please refer to our [Orange's colors]() section and to [Color]() guidelines on  the Orange Design System website.

Please refer to the [Forms]() guidelines and the [Pages]() form examples on the Orange Design System website.

For the correct appearance, refer to the Boosted [Dropdowns]({{< docsref "/components/dropdowns" >}}) component. Be aware that this Dropdowns component, because using an `<ul>` tag, can not be a replacement here for technical and accessibility reasons. - DO NOT EXIST ANYMORE
---
General rule proposition:
When the look is not the good one: use "guidelines" or "design specifications"
When the behavior is not good, use: "guidelines"